Recurly/Resource.cs

```diff
@@ -5,6 +5,17 @@ namespace Recurly
     [ExcludeFromCodeCoverage]
     public class Resource
     {
+        private Response _response;
+
+        public Response GetResponse()
+        {
+            return _response;
+        }
+
+        public void SetResponse(Response response)
```

Comment:
I wonder if this should be `internal` rather than `public`. Perhaps we want to let people set it for testing purposes?

Reply:
Discussing offline.